hack/local-up-cluster.sh

```diff
@@ -253,6 +253,9 @@ if [[ ${CONTAINER_RUNTIME} == "docker" ]]; then
     CGROUP_DRIVER=$(docker info | grep "Cgroup Driver:" | cut -f3- -d' ')
     echo "Kubelet cgroup driver defaulted to use: ${CGROUP_DRIVER}"
   fi
+  if [[ -f /var/log/docker.log && ! -f ${LOG_DIR}/docker.log ]]; then
```

Comment:
Are we certain $LOG_DIR exists?

Reply:
following the pattern like the rest of the file. if LOG_DIR is not set we default to `/tmp` and if it is set we assume it's created already